hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlTestSupport.java

```diff
@@ -218,4 +230,23 @@ public static List<Row> jdbcRows(String query, String connectionUrl) {
             throw new RuntimeException(e);
         }
     }
+    public static List<Row> jdbcRows(String query, String connectionUrl, List<Class<?>> columnType) {
+        List<Row> rows = new ArrayList<>();
+        try (Connection conn = DriverManager.getConnection(connectionUrl);
+             Statement stmt = conn.createStatement()
+        ) {
+            stmt.execute(query);
+            ResultSet resultSet = stmt.getResultSet();
+            while (resultSet.next()) {
+                Object[] values = new Object[resultSet.getMetaData().getColumnCount()];
+                for (int i = 0; i < values.length; i++) {
+                    values[i] = resultSet.getObject(i + 1, columnType.get(i));
```

Comment:
Does this accept Object.class? Maybe we could use that if `columnType` is null, and remove the very similar method above.

Reply:
Compiler doesn't complain when I write `resultSet.getObject(i` + 1, Object.class);`

Reply:
This is what I had in mind 
1dd14649fe2